core/SConscript.kernel

```diff
@@ -206,6 +206,7 @@ SOURCE_MOD += [
     'embed/util/translations/translations.c',
     'embed/util/image/image.c',
     'embed/util/rsod/rsod.c',
+    'embed/util/scm_revision/scm_revision.c',
```

Comment:
we probably can't do this - scm revision is part of authenticity check and its supposed to be hard to find in binary so that fake bootloader/firmware cannot pretend new firmware was installed by providing correct scm revision. in firmware, the object files are shuffled so that the location of scm revision is randomized, but no such thing is done here, even if it was we might not want to duplicate the string as might make finding it easier. Applies to secmon too.